### ISSUE TITLE
Command to deal with channels

### ISSUE THREAD

**Issue by shubhamJay** [coursier/coursier#1786]
Adding and using channels feels a bit confusing now as some commands support `—add-channel`  option, and some don’t. Even though the once which does not allow adding channels, use the channels which are previously added by others commands. For example. `launch` command does not support `—add-channel` option, but it uses channels added by others commands such as `install` command.

  May be having a dedicated command like `channel` which will allow operations such as add channel or remove channel (Not sure that remove channel exists)
Something like: 
```
 cs channel —add <channel> 
cs channel —remove <channel>
```

**Comment by alexarchambault** [comment]
@shubhamJay I agree a `channel` command would be a nice addition, and a better substitute to the `--add-channel` option of `install`.

PRs welcome! https://github.com/coursier/coursier/pull/1780 adds an extra command, as an example.

**Comment by kubukoz** [comment]
Looks like `channel --add` already exists, so this issue is now limited to a missing `--remove`, right?
